twitchio/websocket.py

```diff
@@ -119,6 +132,8 @@ async def _connect(self):
 
         if self._keeper:
             self._keeper.cancel()  # Stop our current keep alive.
+        if self._task_cleaner:
+            self._task_cleaner.cancel()  # Stop our current task cleaner.
```

Comment:
this doesn't really need to ever be cancelled except for at shutdown

Reply:
That's right, I will adapt that part.